dts/arm/st/c0/stm32c0.dtsi

```diff
@@ -67,6 +67,23 @@
 		};
 	};
 
+	mcos {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		mco1: mco@1 {
```

Comment:
@erwango After the last code changes the @1 and reg =<1> is no longer needed. (only needed to macro concat before).

So question is if we should keep it or not?